04 - Guides, Workflows, & Courses/for Theme Designers.md

```diff
@@ -37,6 +37,7 @@ This note collects resources and guides for beginner and expert theme designers
 ## Tools
 - [**Obsidian Theme Template**](https://github.com/obsidian-community/obsidian-theme-template) A GitHub repository template with everything you need to get started. Click `Use this template` to get a ready-to-go theme repository.
 - [**Theme Dev Vault**](https://github.com/obsidian-community/theme-dev-vault) An obsidian vault to aid in theme development
+- [obsidian-theme-design-utilities|Theme Design Utilities Plugin]
```

Comment:
Is this supposed to link to the plugin in the hub? I think it's missing another set of brackets around it `[[ ]]`

Reply:
you are right lol
think I forgot to add them after changing the external link to an internal one